### PR TITLE
Add aesthetic changes to the UG

### LINKED ISSUE #138: Possibly replace bullet points with other aesthetic elements
Separate commands into sections
The colorful extra info

### PR DESCRIPTION
**Pull request by PohSayKeong**
Closes #138

### CODE REVIEW

**Review by lshaoqin — APPROVED** [review]
Merging first for ease of review

File: docs/UserGuide.md [hunk 1]
```diff
@@ -61,35 +61,31 @@ Teaching Assistant Manager (TAManager) is a desktop application for managing tea
 </div>
 
 <span style="float:right; font-size: 0.8em;">[BACK TO TOP](#table-of-contents)</span>
```
Comment (lshaoqin) [inline docs/UserGuide.md:63]: I think it would be good to add a line here

File: docs/UserGuide.md [hunk 2]
```diff
@@ -61,35 +61,31 @@ Teaching Assistant Manager (TAManager) is a desktop application for managing tea
...
-### Viewing help : `help`
-
-Shows a message explaining how to access the help page.
-
-Format: `help`
 
-![help message](images/helpMessage.png)
+## Teaching Assistant Commands
```
Comment (lshaoqin) [inline docs/UserGuide.md:65]: TA Management Commands?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -61,35 +61,31 @@ Teaching Assistant Manager (TAManager) is a desktop application for managing tea
...
+- `HOUR` should be an integer.
 - `TAG` should be an alphanumeric string without spaces.
 - `COURSE_CODE` should start with 2-3 alphabets, followed by 4 numbers, and optionally end with an alphabet.
-- `HOUR` should be an integer.
 
 Example:
-`add n/ Rayner Toh p/93812311 e/rayner@example.com tele/@raynertjx t/parttime c/CS2103T h/4` will add a new teaching assistant named Rayner Toh to TAManager.
+`add n/ Rayner Toh p/93812311 e/rayner@example.com tele/@raynertjx h/4 t/parttime c/CS2103T` will add a new teaching assistant named Rayner Toh to TAManager.
 
 ![add TA](images/addTA.png)
```
Comment (lshaoqin) [inline docs/UserGuide.md:84]: Would be good to update this screenshot

File: docs/UserGuide.md [hunk 4]
```diff
@@ -295,16 +309,23 @@ Examples:
...
 
-When the command fails:
+<div markdown="block" class="alert alert-danger">
+:x: When the command fails:
 
 - Incorrect format (missing prefix or parameter, or `DAY` out of range, or invalid `FROM` or `TO` format): `Invalid command format!`
 - Index out of range: `The person index provided is invalid`
+</div>
+
+## Course Commands
```
Comment (lshaoqin) [inline docs/UserGuide.md:328]: Course Management Commands

File: docs/UserGuide.md [hunk 5]
```diff
@@ -382,11 +425,11 @@ Exits the program.
 
 Format: `exit`
 
-### Saving the data
+## Saving the data
```
Comment (lshaoqin) [inline docs/UserGuide.md:428]: If there is no requirement in the rubrics, I think we can move these under the FAQ.

**Review by lshaoqin — COMMENTED** [review]
Thanks for the great work! Regarding the colours, I think there are too many things jumping out at the reader at the moment. Let's discuss this further during the meeting.

File: docs/UserGuide.md (hunk 3 above)
Comment (PohSayKeong) [reply docs/UserGuide.md:84]: is this screenshot outdated?

Comment (lshaoqin) [reply docs/UserGuide.md:84]: I think so, new TAs don't have free timeslots

**Pull request merged**